src/server/clientiface.cpp

```diff
@@ -147,14 +148,15 @@ void RemoteClient::GetNextBlocks (
 	u16 max_simul_sends_usually = m_max_simul_sends;
 
 	/*
-		Check the time from last addNode/removeNode.
-
 		Decrease send rate if player is building stuff.
+
+		The idea is that we can save some bandwidth since the player is busy
+		and not looking around.
 	*/
 	m_time_from_building += dtime;
 	if (m_time_from_building < m_min_time_from_building) {
-		max_simul_sends_usually
-			= LIMITED_MAX_SIMULTANEOUS_BLOCK_SENDS;
+		max_simul_sends_usually *= LIMITED_BLOCK_SENDS_FACTOR;
```

Comment:
Interesting. For larger viewing_ranges (or actual servers with some latency to the client) `max_simultaneous_block_sends_per_client`  has to be increased significantly. For 1000-2000 I have it set to 512, for example.